packages/network/src/network.js

```diff
@@ -10,15 +10,28 @@ import { Shape } from './shapes.js';
 
 /// <reference path="./types.js" />
 /**
- * @import {AttemptDescription, Bytes, Closable, CloseReason, Connection, ConnectionHandler, Endpoint, ListenHandler, Port, Protocol, ProtocolHandler, ProtocolImpl} from './types.js';
+ * @import {
+ *  AttemptDescription, Bytes, CloseReason, Closable,
+ *  Connection, ConnectionHandler, Endpoint, ListenHandler,
+ *  Port, Protocol, ProtocolHandler, ProtocolImpl
+ * } from './types.js';
+ * @import { PromiseVow, Remote, VowTools } from '@agoric/vow';
  */
```

Comment:
Do you think that the `*` at the beginning of each line within the JSDoc comment block might be causing the issue? When I removed these asterisks, the `network.d.ts` file generated without any problems related to the types.

After removing the asterisks:
```js
/**
@import {
 AttemptDescription, Bytes, CloseReason, Closable,
 Connection, ConnectionHandler, Endpoint, ListenHandler,
 Port, Protocol, ProtocolHandler, ProtocolImpl
} from './types.js';
@import { PromiseVow, Remote, VowTools } from '@agoric/vow';
*/

```

Reply:
After removing the asterisks, the error message now indicates issues with non-existent module imports:
```bash
test/devices.js(1,27): error TS2307: Cannot find module '../bundles/bundle-centralSupply.js' or its corresponding type declarations.
test/devices.js(2,24): error TS2307: Cannot find module '../bundles/bundle-mintHolder.js' or its corresponding type declarations.
test/devices.js(3,27): error TS2307: Cannot find module '../bundles/bundle-provisionPool.js' or its corresponding type declarations.

```

These errors are occurring in the `packages/vats/test/devices.js` file. Trying to understand these.